ggshield/utils/git_shell.py

```diff
@@ -204,7 +204,7 @@ def git(
     try:
         logger.debug("command=%s timeout=%d", command, timeout)
         result = subprocess.run(
-            [_get_git_path()] + command,
+            [_get_git_path()] + ["-c", "core.quotePath=false"] + command,
```

Comment:
Can you add a comment explaining why we set `core.quotePath` to false?

nit, the two lists can be merged:
```suggestion
            [_get_git_path(), "-c", "core.quotePath=false"] + command,
```